doc/api/packages.md

```diff
@@ -683,6 +683,32 @@ and in a CommonJS one. For example, this code will also work:
 const { something } = require('a-package/foo'); // Loads from ./foo.js.
 ```
 
+Finally, self-referencing also works with scoped packages. For example, this
+code will also work:
+
+```json
+// package.json
+{
+  "name": "@my/package",
+  "exports": "./index.js"
+}
+```
+
+```js
+// ./index.js
+module.exports = 42;
+```
+
+```js
+// ./other.js
+console.log(require('@my/package'));
+```
+
+```bash
+node other.js
+# 42
```

Comment:
````suggestion
```console
> node other.js
42
````

Reply:
Can you explain that? I copied the bash code blocks from other places. Can you point me to a console code block?

Reply:
Sure, here's one: https://github.com/nodejs/node/blob/674614b3f5245f1b5cae974af1365fd632190b41/doc/api/modules.md#L343-L353

Reply:
Great, fixed :-)

There are 100 files that are using the `bash` language in code blocks, maybe also related to `npm` or other dependencies. Should we change the Node.js ones in another PR, or is it ok to left them as they are currently?

Reply:
Here's how we use those in Node.js:
`bash` is a valid language code block as long as it describes a list of instructions or an actual shell script.
`console` is more relevant when you want to show both the command and its output.

We usually don't touch the files from our dependencies, it's OK if they use different conventions than ours; those can stay as is. For Node.js ones, if you see somewhere the wrong language block is being used, of course a PR is welcome :)

Reply:
> `bash` is a valid language code block as long as it describes a list of instructions or an actual shell script.
> `console` is more relevant when you want to show both the command and its output.

Great, thanks for the clarification, makes sense :-)


> We usually don't touch the files from our dependencies, it's OK if they use different conventions than ours; those can stay as is.

Obviously, I would not touch the ones for the dependencies, because they would be overwritting when updating them... :-)